src/server.rs

```diff
@@ -69,6 +71,84 @@ fn replace_string_in_range(
     contents
 }
 
+fn function_defines(name: String, f: &FunctionExpr) -> bool {
+    for param in f.params.clone() {
+        if param.key.name == name {
+            return true;
+        }
+    }
+
+    false
+}
+
+fn is_scope(name: String, n: Rc<Node<'_>>) -> bool {
+    let mut dvisitor: DefinitionFinderVisitor =
```

Comment:
Is this explicit type needed here? I would think it's rather obvious what the result of `DefinitionFinderVisitor::now` is to the compiler.